packages/graphql/tests/tck/directives/auth/projection.test.ts

```diff
@@ -104,27 +104,27 @@ describe("Cypher Auth Projection", () => {
         });
 
         expect(formatCypher(result.cypher)).toMatchInlineSnapshot(`
-            "CALL {
-            CREATE (this0:User)
-            SET this0.id = $this0_id
-            RETURN this0
-            }
+            "UNWIND $create_param0 AS create_var1
             CALL {
-            CREATE (this1:User)
-            SET this1.id = $this1_id
-            RETURN this1
+                WITH create_var1
+                CREATE (create_this0:\`User\`)
+                SET
+                    create_this0.id = create_var1.id
+                RETURN create_this0
             }
-            CALL apoc.util.validate(NOT ((this0.id IS NOT NULL AND this0.id = $projectionauth_param0)), \\"@neo4j/graphql/FORBIDDEN\\", [0])
```

Comment:
This seems to be previously wrong as `Allow` rules should not have any effect on the `CREATE` operation,
For reference: https://neo4j.com/docs/graphql-manual/current/auth/authorization/allow/